src/pages/pos-pointOfSales/forms/UsersForm.js

```diff
@@ -0,0 +1,169 @@
+import { useFormik } from 'formik'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import FormShell from 'src/components/Shared/FormShell'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useContext, useEffect } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { PointofSaleRepository } from 'src/repositories/PointofSaleRepository'
+import { SaleRepository } from 'src/repositories/SaleRepository'
+
+const UsersForm = ({ store, labels, maxAccess, editMode }) => {
```

Comment:
editMode props not passed from window, remove it and declare it inside this form